docs/src/administration/web/configure-environment.md

```diff
@@ -76,7 +76,7 @@ the number of active workers and cron jobs, and the command to SSH into the cont
 
 You can also see all cron jobs with their name, frequency, and command.
 
-![Services: app overview](/images/management-console/service-tab/app-overview.png "0.7")
+![Services: app overview](/images/management-console/service-tab/app-overview.png "0.5")
```

Comment:
💅🏼  I think [the screenshot](https://pr-2237-xexrsbi-652soceglkw4u.eu-3.platformsh.site//administration/web/configure-environment.html) could be cut ["more nicely"](https://github.com/platformsh/platformsh-docs/blob/main/contributing/content-style.md#format-screenshots-for-sustainability), to include the full line of redacted ssh access.

Reply:
I don't think the style guide says it should have more in there. It says "Crop the screenshot so it only shows what users need to get the context". I think the context is clear from the shot, just not the details.

What benefit do you think showing that part brings? I was thinking of it as details that might distract people from the point of the page (and also make the rest of it smaller), but I could very well be missing something.

Reply:
It's just for a style perspective of things, so it's safe to ignore ;)

>  I was thinking of it as details that might distract people from the point of the page (and also make the rest of it smaller)
This is correct, and that's why I proposed to discuss it.

Cutting the screenshot "nicely" doesn't add much more clarity but it makes the page look a bit nicer, that's all.